static/js/rdm.js

```diff
@@ -1,2 +1,838 @@
-angular.module("rdmApp",[]).config(["$interpolateProvider",function(a){"use strict";a.startSymbol("{[{"),a.endSymbol("}]}")}]).controller("UIDController",["$scope",function(a){"use strict";a.invalid_input_message="Invalid UID, please enter a UID in the form MMMM:NNNNNNNN",a.euid="",a.error="",a.uid="",a.convertToEUID=function(){a.euid="",a.error="";var b=a.uid.replace(/[\.:\-\s]/g,"");if(!b.match(/^[0-9a-fA-F]{12}$/))return void(a.error=a.invalid_input_message);var c=b.slice(0,4),d=b.slice(4,12);if(4!==c.length||8!==d.length)return void(a.error=a.invalid_input_message);var e=parseInt(c,16),f=parseInt(d,16);if(isNaN(e)||isNaN(f))return void(a.error=a.invalid_input_message);var g=[],h=e>>8&255,i=255&e,j=f>>24&255,k=f>>16&255,l=f>>8&255,m=255&f;g.push(170|h),g.push(85|h),g.push(170|i),g.push(85|i),g.push(170|j),g.push(85|j),g.push(170|k),g.push(85|k),g.push(170|l),g.push(85|l),g.push(170|m),g.push(85|m);var n=g.reduce(function(a,b,c,d){return a+b}),o=n>>8&255,p=255&n;g.push(170|o),g.push(85|o),g.push(170|p),g.push(85|p),a.euid=g.map(function(a){return a.toString(16)}).join(" ")}}]).controller("EUIDController",["$scope",function(a){"use strict";a.euid="",a.error="",a.uid="",a.convertToUID=function(){function b(a,b){var c=a.toString(16),d=b-c.length+1;return new Array(+(d>0&&d)).join("0")+c}a.error="",a.uid="";var c=a.euid.replace(/[,\s]/g,"").trim();if(!c.match(/^[0-9a-fA-F]*$/))return void(a.error="Invalid EUID: non hex characters");if(32!==c.length)return void(a.error="Invalid EUID: insufficent data, should be 32 hex characters");for(var d=[],e=0;16>e;e++){var f=c.slice(2*e,2*(e+1)),g=parseInt(f,16);if(isNaN(g))return void(a.error="Invalid EUID: bad value"+f);d.push(g)}var h=d[0]&d[1],i=d[2]&d[3],j=(h<<8)+i,k=d[4]&d[5],l=d[6]&d[7],m=d[8]&d[9],n=d[10]&d[11],o=(k<<24)+(l<<16)+(m<<8)+n,p=((d[12]&d[13])<<8)+(d[14]&d[15]),q=d.slice(0,12).reduce(function(a,b,c,d){return a+b});return p!==q?void(a.error="Checksum mismatch, was "+p+", calculated to be "+q):void(a.uid=b(j)+":"+b(o,8))}}]);
-//# sourceMappingURL=rdm.js.map
\ No newline at end of file
+angular.module('rdmApp', [])
```

Comment:
No, I'm not really sure why we're checking in the minified versions. I think I'll remove them in another change.